business-registry-dashboard/app/components/Table/AffiliatedEntity/Action.vue

```diff
@@ -1,64 +1,74 @@
 <script setup lang='ts'>
 import type { DropdownItem } from '#ui/types'
 import {
-  NrRequestActionCodes
-  // FilingTypes
+  NrRequestActionCodes,
+  FilingTypes
   // NrRequestTypeCodes
 } from '@bcrs-shared-components/enums'
 // import AffiliationInvitationService from '@/services/affiliation-invitation.services'
 // import launchdarklyServices from 'sbc-common-components/src/services/launchdarkly.services'
 
 const props = defineProps<{
+  affiliations: Business[],
```

Comment:
add prop so action component doesnt use useAffiliation composable and trigger multiple fetch requests